Dockerfile

```diff
@@ -87,53 +87,81 @@ RUN wget https://github.com/jakzal/toolbox/releases/download/v$TOOLBOX_VERSION/t
 # PHP with all extensions needed, correctly configured
 FROM php-base as php-configured
 
-# Extensions .so
+# Install extensions + extension config
 COPY --link --from=ast-builder /usr/local/lib/php/extensions /usr/local/lib/php/extensions
-COPY --link --from=pcov-builder /usr/local/lib/php/extensions /usr/local/lib/php/extensions
-COPY --link --from=extensions-builder /usr/local/lib/php/extensions /usr/local/lib/php/extensions
-
-# Extensions .ini
 COPY --link --from=ast-builder /usr/local/etc/php/conf.d /usr/local/etc/php/conf.d
+COPY --link --from=pcov-builder /usr/local/lib/php/extensions /usr/local/lib/php/extensions
 COPY --link --from=pcov-builder /usr/local/etc/php/conf.d  /usr/local/etc/php/conf.d
+COPY --link --from=extensions-builder /usr/local/lib/php/extensions /usr/local/lib/php/extensions
 COPY --link --from=extensions-builder /usr/local/etc/php/conf.d /usr/local/etc/php/conf.d
 
+# Base php.ini
 COPY --link <<EOF /usr/local/etc/php/conf.d/phpqa.ini
 date.timezone=Europe/London
 memory_limit=-1
 phar.readonly=0
 pcov.enabled=0
 EOF
 
-COPY --link --chmod=755 <<EOF /entrypoint.sh
-#!/usr/bin/env sh
-set -e
+# Validate the PHP configuration
+RUN php -m | grep ast
```

Comment:
I can't think how to make this a one-liner

Reply:
Not sure if better:

```
php -m | tr '\n' ' ' | awk '/ast/ && /pcov/ && /bcmath/ && /intl/ && /zip/ && /bz2/'
```